### DISCUSSION TITLE
Collapsible content block support

### DISCUSSION THREAD

**Discussion by benhowes** [outline/outline#2312] [Ideas]
Sometimes it's really handy to be able to hide away some detailed explanation without interrupting the main flow of the document. In github markdown I'd be able to use `<details>` + `<summary>` tags to add that in ([Overview of that here if you're not familiar](https://gist.github.com/scmx/eca72d44afee0113ceb0349dd54a84a2))

Would it be possible to get some similar functionality in outline?

**Comment by tommoor** [comment]
We'd like to bring collapsing of content back, it previously existed on headings and I think this is a reasonable way to go – when we moved editor framework last year the functionality was lost. See ticket: 

https://github.com/outline/outline/issues/1338

**Reply by benhowes** [reply]
Thanks @tommoor

**Reply by aam1r** [reply]
This just came up internally. We'd love to have this back so we can collapse code blocks 🙂

**Reply by tommoor** [reply]
It's supported on headings again now – so the workaround is to put a heading above your code block. I'm not sure if we'll ever support collapsing all types of nodes

**Reply by lookto** [reply]
This is not realy a nice workaround. Truncating/Collapsing of code blocks would be realy nice to have.
